docs/source/science/NISAR/NISAR_access.ipynb

```diff
@@ -2,23 +2,23 @@
  "cells": [
```

Comment:
<p>same comment as above - Should we try to make this relative?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>